optional_plugins/varianter_yaml_to_mux/tests/test_mux.py

```diff
@@ -127,10 +127,10 @@ def test_basic_functions(self):
                   'prod']
         self.assertEqual(leaves, self.tree.get_leaves())
         # ascii contain all leaves and doesn't raise any exceptions
-        ascii = tree.tree_view(self.tree, 0, False).decode('ascii')
+        ascii_ = tree.tree_view(self.tree, 0, False).decode('ascii')
```

Comment:
I know it's just selftest, but using `${variable}_` is not really nice practice. How about about `tree_view` or just `output`?